soh/src/code/z_onepointdemo.c

```diff
@@ -330,12 +353,20 @@ s32 OnePointCutscene_SetInfo(PlayState* play, s16 camIdx, s16 csId, Actor* actor
         case 9601:
             Play_CameraChangeSetting(play, camIdx, CAM_SET_CS_3);
             Play_CameraChangeSetting(play, MAIN_CAM, mainCam->prevSetting);
-            OnePointCutscene_SetCsCamPoints(csCam, D_80120430 | 0x1000, D_8012042C, D_80120308, D_80120398);
+            if (CVarGetInteger("gCrawlSpeed", 0) >= 2) {
```

Comment:
These two if blocks aren't necessary.  With my suggestions above, you've already ensured that the crawl timer will always be initialized to the default value of 90 and will only be not 90 when the slider is above 1, and that D_8012042C constant is just 90, so you can just unconditionally call this function with the crawl timer, no else condition needed.

Reply:
This one was intentional to preserve the original code. Probably okay, since the D_8012042C  constant is now used to initialize the variable above?

Reply:
I re-organized this entirely to conform to the proposed vanilla code guidelines, with some duplication to retain clarity.